client/rpc/dex_client.go

```diff
@@ -95,13 +97,7 @@ func (c *HTTP) GetBalances(addr types.AccAddress) ([]types.TokenBalance, error)
 	var denoms map[string]bool
 	denoms = map[string]bool{}
 	for _, coin := range coins {
-		denom := coin.Denom
-		exists := c.existsCC(denom)
-		// TODO: we probably actually want to show zero balances.
-		// if exists && !sdk.Int.IsZero(coins.AmountOf(denom)) {
-		if exists {
-			denoms[denom] = true
-		}
+		denoms[coin.Denom] = true
```

Comment:
Can we just iterate `coins` directly and remove `denoms`?

Reply:
sure. changed